modules/apps/object/object-rest-test/src/testFunctional/tests/FilterCustomObjectsMultipleLevelsRelationshipsWithStringFunctions.testcase

```diff
@@ -197,4 +317,334 @@ definition {
+				property = "$.items[0].name",
+				response = ${response});
+
+			TestUtils.assertEquals(
+				actual = ${actual},
+				expected = "Liferay");
+
+			HeadlessWebcontentAPI.assertProperNumberOfItems(
+				expectedTotalElement = 1,
+				responseToParse = ${response});
+		}
+	}
+
+	@disable-webdriver = "true"
+	@priority = 3
+	test CanGetNoFilteredItemsWhenFilteringInStudentsEndpointByStringFunctions {
+		task ("And Given a oneToMany relationship universityStudents") {
+			ObjectDefinitionAPI.createRelationshipWithObjectDefinitionNames(
+				childObjectName = "Student",
+				deletionType = "cascade",
+				name = "universityStudents",
+				parentObjectName = "University",
+				type = "oneToMany");
+		}
+
+		task ("And Given a manyToMany relationship subjectsStudents") {
+			ObjectDefinitionAPI.createRelationshipWithObjectDefinitionNames(
+				childObjectName = "Student",
+				deletionType = "cascade",
+				name = "subjectsStudents",
+				parentObjectName = "Subject",
+				type = "manyToMany");
+		}
+
+		task ("And Given student 'Tom' related to the university 'Liferay' through universities PUT endpoint") {
+			ObjectDefinitionAPI.relateObjectEntriesByName(
+				child_plural_label = "students",
+				childEntryName = "Tom",
+				parent_plural_label = "universities",
+				parentEntryName = "Liferay",
+				relationshipName = "universityStudents");
+		}
+
+		task ("And Given I relate student entry 'Tom' to subject entry 'Math' through subjects PUT endpoint") {
+			ObjectDefinitionAPI.relateObjectEntriesByName(
+				child_plural_label = "students",
+				childEntryName = "Tom",
+				parent_plural_label = "subjects",
+				parentEntryName = "Math",
+				relationshipName = "subjectsStudents");
+		}
+
+		task ("When using get request o/c/students?filter=startswith(subjectsStudents/universityStudents/country, 'Spa') or contains(subjectsStudents/universityStudents/country, 'ain')") {
+			var responseFromStartsWith = ObjectDefinitionAPI.getObjectEntries(
+				en_US_plural_label = "students",
+				parameter = "filter=startswith%28subjectsStudents%2FuniversityStudents%2Fcountry%2C%20%27Spa%27%29");
+			var responseFromContains = ObjectDefinitionAPI.getObjectEntries(
+				en_US_plural_label = "students",
+				parameter = "filter=contains%28subjectsStudents%2FuniversityStudents%2Fcountry%2C%20%27ain%27%29");
+		}
+
+		task ("Then the response contains the university entry details") {
+			var actualFromStartsWith = JSONPathUtil.getProperty(
+				property = "$.title",
+				response = ${responseFromStartsWith});
+			var actualFromContains = JSONPathUtil.getProperty(
+				property = "$.title",
+				response = ${responseFromContains});
+
+			TestUtils.assertPartialEquals(
+				actual = ${actualFromStartsWith},
+				expected = "A property used in the filter criteria is not supported");
+
+			TestUtils.assertPartialEquals(
+				actual = ${actualFromContains},
+				expected = "A property used in the filter criteria is not supported");
```

Comment:
The expected message is NotSupported, but the test name is CanGetNoFilteredItems***, could you check and update?

Reply:
the ticket and the test map have different names, I will check and modify it